samples/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+# Add all samples
+add_subdirectory(blink)
```

Comment:
Not an action item for this PR, but something convenient for the future - create a simple shell script that auto-generates a new CMakeLists.txt based on a template and appends the `add_subdirectory` in the appropriate location(s) every time you want to make a new executable

Reply:
I'll look into how to do that in CMake, I shy away from making a shell script since that takes away the Cross Platformness of CMake.